### PR TITLE
Apply `yapf` to `pl_bolts` 1/2

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Follow-up PR of #509 and part of #482.

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by akihironitta** [comment]
It looks `yapf` doesn't care long-name objects, so I placed `noqa: E501` to ignore flake8 for better readability in https://github.com/PyTorchLightning/pytorch-lightning-bolts/pull/510/commits/51154d20a543b49e14d4ec03a9e03299f68e6ce2

**Comment by Borda** [comment]
> It looks `yapf` doesn't care long-name objects, so I placed `noqa: E501` to ignore flake8 for better readability in [51154d2](https://github.com/PyTorchLightning/pytorch-lightning-bolts/commit/51154d20a543b49e14d4ec03a9e03299f68e6ce2). I looked through the yapf docs, but they don't have any option to format this.

this may resolve the case `ALLOW_SPLIT_BEFORE_DICT_VALUE = true`

**Comment by akihironitta** [comment]
> > It looks `yapf` doesn't care long-name objects, so I placed `noqa: E501` to ignore flake8 for better readability in [51154d2](https://github.com/PyTorchLightning/pytorch-lightning-bolts/commit/51154d20a543b49e14d4ec03a9e03299f68e6ce2). I looked through the yapf docs, but they don't have any option to format this.
> 
> this may resolve the case `ALLOW_SPLIT_BEFORE_DICT_VALUE = true`

You're right, but it can lower readability a lot in other places:
```python
--- a/pl_bolts/models/self_supervised/amdim/amdim_module.py
+++ b/pl_bolts/models/self_supervised/amdim/amdim_module.py
@@ -239,13 +239,20 @@ class AMDIM(pl.LightningModule):
         # CIFAR 10
         cf_root_lr = 2e-4
         cifar_10 = {
-            'dataset': 'cifar10',
-            'ndf': 320,
-            'n_rkhs': 1280,
-            'depth': 10,
-            'image_height': 32,
-            'batch_size': 200,
-            'nb_classes': 10,
+            'dataset':
+            'cifar10',
+            'ndf':
+            320,
+            'n_rkhs':
+            1280,
+            'depth':
+            10,
+            'image_height':
+            32,
+            'batch_size':
+            200,
+            'nb_classes':
+            10,

```

so, shall we go with `noqa: E501`?

File: pl_bolts/models/self_supervised/resnets.py [hunk 1]
```diff
@@ -158,19 +167,17 @@ def __init__(
             # the 2x2 stride with a dilated convolution instead
             replace_stride_with_dilation = [False, False, False]
         if len(replace_stride_with_dilation) != 3:
-            raise ValueError("replace_stride_with_dilation should be None "
-                             "or a 3-element tuple, got {}".format(replace_stride_with_dilation))
+            raise ValueError(
+                "replace_stride_with_dilation should be None "
+                "or a 3-element tuple, got {}".format(replace_stride_with_dilation)
```
Comment (Borda) [inline pl_bolts/models/self_supervised/resnets.py:172]:
```suggestion
                f"or a 3-element tuple, got {replace_stride_with_dilation}"
```

File: pl_bolts/models/self_supervised/resnets.py [hunk 2]
```diff
@@ -221,13 +225,23 @@ def _make_layer(self, block, planes, blocks, stride=1, dilate=False):
             )
 
         layers = []
-        layers.append(block(self.inplanes, planes, stride, downsample, self.groups,
-                            self.base_width, previous_dilation, norm_layer))
+        layers.append(
+            block(
+                self.inplanes, planes, stride, downsample, self.groups, self.base_width, previous_dilation, norm_layer
```
Comment (Borda) [inline pl_bolts/models/self_supervised/resnets.py:230]:
```suggestion
                self.inplanes, planes, stride, downsample, self.groups, self.base_width, previous_dilation, norm_layer,
```

File: pl_bolts/models/self_supervised/simclr/simclr_module.py [hunk 3]
```diff
@@ -207,33 +202,18 @@ def exclude_from_wt_decay(self, named_params, weight_decay, skip_list=['bias', '
             else:
                 params.append(param)
 
-        return [
-            {'params': params, 'weight_decay': weight_decay},
-            {'params': excluded_params, 'weight_decay': 0.}
-        ]
+        return [{'params': params, 'weight_decay': weight_decay}, {'params': excluded_params, 'weight_decay': 0.}]
```
Comment (Borda) [inline pl_bolts/models/self_supervised/simclr/simclr_module.py:205]:
```suggestion
        return [{'params': params, 'weight_decay': weight_decay}, {'params': excluded_params, 'weight_decay': 0.,}]
```

File: pl_bolts/models/self_supervised/simclr/transforms.py [hunk 4]
```diff
@@ -153,9 +143,7 @@ def __init__(
         self.normalize = normalize
 
         self.color_jitter = transforms.ColorJitter(
-            0.8 * self.jitter_strength,
-            0.8 * self.jitter_strength,
-            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength, 0.8 * self.jitter_strength, 0.8 * self.jitter_strength,
             0.2 * self.jitter_strength
```
Comment (Borda) [inline pl_bolts/models/self_supervised/simclr/transforms.py:147]:
```suggestion
            0.2 * self.jitter_strength,
```

File: pl_bolts/models/self_supervised/swav/transforms.py [hunk 5]
```diff
@@ -146,9 +139,7 @@ def __init__(
         self.normalize = normalize
 
         self.color_jitter = transforms.ColorJitter(
-            0.8 * self.jitter_strength,
-            0.8 * self.jitter_strength,
-            0.8 * self.jitter_strength,
+            0.8 * self.jitter_strength, 0.8 * self.jitter_strength, 0.8 * self.jitter_strength,
             0.2 * self.jitter_strength
```
Comment (Borda) [inline pl_bolts/models/self_supervised/swav/transforms.py:143]:
```suggestion
            0.2 * self.jitter_strength,
```

File: pl_bolts/transforms/dataset_normalizations.py [hunk 6]
```diff
@@ -12,8 +12,9 @@ def imagenet_normalization():
 
 
 def cifar10_normalization():
-    normalize = transforms.Normalize(mean=[x / 255.0 for x in [125.3, 123.0, 113.9]],
-                                     std=[x / 255.0 for x in [63.0, 62.1, 66.7]])
+    normalize = transforms.Normalize(
+        mean=[x / 255.0 for x in [125.3, 123.0, 113.9]], std=[x / 255.0 for x in [63.0, 62.1, 66.7]]
```
Comment (Borda) [inline pl_bolts/transforms/dataset_normalizations.py:16]:
```suggestion
        mean=[x / 255.0 for x in [125.3, 123.0, 113.9]], std=[x / 255.0 for x in [63.0, 62.1, 66.7]],
```

**Comment by Borda** [comment]
> You're right, but it can lower readability a lot in other places:

ok, so just keep it as it is now :]

**Review by Borda — APPROVED** [review]

**Pull request merged**